lib-nearby/src/main/java/com/meshrabiya/lib_nearby/nearby/NearbyStreamHeader.kt

```diff
@@ -0,0 +1,28 @@
+package com.meshrabiya.lib_nearby.nearby
+
+import java.nio.ByteBuffer
+
+data class NearbyStreamHeader(
```

Comment:
This should extend MmcpMessage - we also need the info included in it / virtual packet e.g. from address, to address, etc.